lib/js/src/manager/screen/choiceset/_ChoiceSetManagerBase.js

```diff
@@ -508,14 +516,46 @@ class _ChoiceSetManagerBase extends _SubManagerBase {
 
     /**
      * Defauly keyboard properties object
-     * @returns {KeyboardProperties}
+     * @returns {KeyboardProperties} - A KeyboardProperties RPC
      */
     _defaultKeyboardConfiguration () {
         return new KeyboardProperties()
             .setLanguage(Language.EN_US)
             .setKeyboardLayout(KeyboardLayout.QWERTY)
             .setKeypressMode(KeypressMode.RESEND_CURRENT_ENTRY);
     }
+
+    /**
+     * Listen for DISPLAYS capability updates
+     * @private
+     */
+    _addListeners () {
```

Comment:
This method looks to be the same as the `_SubManagerBase._handleDisplayCapabilities` method.

Reply:
Line 538 has the small difference. The display name from the system capabilities needs to be taken and saved as displayName in this class